docs/command_line_tool/phylum_extension_run.md

```diff
@@ -0,0 +1,29 @@
+---
+title: phylum extension run
+category: 6255e67693d5200013b1fa3e
+hidden: true
+---
+
+Run an extension from a directory
+
+```sh
+phylum extension run [OPTIONS] <PATH> [OPTIONS]...
+```
+
+### Options
+
+`-y`, `--yes`
+&emsp; Automatically accept requested permissions
+
+### Details
+
+The extension will be run without it having been installed first.
```

Comment:
How about:
`The provided extension will be run without prior installation. The provided extension will not be installed. If the extension is already installed, the provided path to the extension is the one that will be run.`

It is wordier, but perhaps more clear. More/other suggestions and tweaks to this language are encouraged.

Reply:
I'm not sure this is more clear, to me it kinda just sounds even more confusing.

The `The provided extension will not be installed` seems somewhat redundant, it reads like you just said the same thing twice.

The last sentence has me all confused about which path of the extension will actually be executed, when in reality it really just doesn't matter if it's installed or not.

I think any focus on installed or not will just confuse people, because the installation status is ultimately irrelevant.